jekyll/_cci2/status-badges.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: classic-docs
+title: Embedding Build Status Badges
+---
+
+## Overview
+
+CircleCI provides build status badges for both projects and branches. Users tend to embed status badges in GitHub READMEs, although they can be placed virtually anywhere.
+
+## Steps
+
+CircleCI provides a tool to generate embed code for status badges.
```

Comment:
add a list of the types we generate.

Reply:
Do we really need this? It would be a literal copy-and-paste from the dropdown menu mentioned in Step 5.